### PR TITLE
Add TAG reader

### PR DESCRIPTION
**Pull request by ozziii**
Some VTO device like VTO2111D-P-S2 have NFC Tag reader. 
With propose code add therse features

### CODE REVIEW

File: custom_components/dahua/__init__.py [hunk 1]
```diff
@@ -355,6 +360,19 @@ def on_receive_vto_event(self, event: dict):
...
+        
+        # OZZIII BEGIN  FIRE TAG READER
+        if code == "AccessControl":
+            card_id = event.get("Data", {}).get("CardNo", "")
+            if card_id:
+                card_id_md5 = hashlib.md5(card_id.encode()).hexdigest()
+                asyncio.run_coroutine_threadsafe(
+                    async_scan_tag(self.hass, card_id_md5, self.get_device_name()), self.hass.loop
+                ).result()
+                return
```
Comment (rroller) [inline custom_components/dahua/__init__.py:372]: Are you certain this return needs to be here?

Comment (jcristovao) [reply custom_components/dahua/__init__.py:372]: I am not the owner of the PR, but I confirm that the code works once you remove the unneeded `return`, as you very well pointed out.

Shall we wait for @ozziii to fix it, or do you want me to do a separate PR?

**Comment by TECH7Fox** [comment]
Should we make a new PR for this? Think @ozziii abandoned this PR. Willing to make a new PR myself.

**Review by ozziii — COMMENTED** [review]
removed return

**Comment by TECH7Fox** [comment]
@rroller could this be merged soon? This feature would be great.

**Review by rroller — APPROVED** [review]

**Comment by rroller** [comment]
Hmm, there's a merge conflict, this needs to be fixed first, then i'll merge

**Comment by rroller** [comment]
I created a new PR and merged it in https://github.com/rroller/dahua/pull/279

**Comment by rroller** [comment]
Released in 0.9.45 - https://github.com/rroller/dahua/releases/tag/0.9.45

**Pull request closed without merging**